### ISSUE TITLE
Show redundancy command not being parsed when last line comes with unavailable

### ISSUE THREAD

**Issue by balmasea** [networktocode/pyntc#222]
New devices with IOS-XE have output for `show redundancy` command that is being wrongly parsed by PyNTC.

```bash
Traceback:
  File \"/var/lib/awx/custom_venvs/gna-a2104p36/lib/python3.6/site-packages/pyntc/devices/ios_device.py\", line 702, in open
    self.confirm_is_active()
  File \"/var/lib/awx/custom_venvs/gna-a2104p36/lib/python3.6/site-packages/pyntc/devices/ios_device.py\", line 407, in confirm_is_active
    if not self.is_active():
  File \"/var/lib/awx/custom_venvs/gna-a2104p36/lib/python3.6/site-packages/pyntc/devices/ios_device.py\", line 652, in is_active
    return self.redundancy_state in self.active_redundancy_states
  File \"/var/lib/awx/custom_venvs/gna-a2104p36/lib/python3.6/site-packages/pyntc/devices/ios_device.py\", line 813, in redundancy_state
    processor_redundancy_info = re_show_redundancy.group(\"self\")
AttributeError: 'NoneType' object has no attribute 'group'
```

Command output:
```bash
#show redundancy
Redundant System Information :
------------------------------
       Available system uptime = 3 years, 27 weeks, 16 minutes
Switchovers system experienced = 0
              Standby failures = 0
        Last switchover reason = none

                 Hardware Mode = Simplex
    Configured Redundancy Mode = sso
     Operating Redundancy Mode = sso
              Maintenance Mode = Disabled
                Communications = Down      Reason: Simplex mode

Current Processor Information :
-------------------------------
               Active Location = slot 5
        Current Software state = ACTIVE
       Uptime in current state = 3 years, 27 weeks, 15 minutes
                 Image Version = Cisco IOS Software, s72033_rp Software (s72033_rp-IPSERVICESK9_WAN-M), Version 12.2(33)SXH6, RELEASE SOFTWARE (fc1)
Technical Support: http://www.cisco.com/techsupport
Copyright (c) 1986-2009 by Cisco Systems, Inc.
Compiled Thu 15-Oct-09 11:30 by prod_rel_team
                          BOOT = sup-bootdisk:s72033-ipservicesk9_wan-mz.122-33.SXH6.bin,12;
        Configuration register = 0x2102

Peer (slot: unavailable) information is not available because it is in 'DISABLED' state
```

Regexp `RE_SHOW_REDUNDANCY` is not working properly.

**Comment by balmasea** [comment]
Reproducing the error I have found that problem is in the `Peer (slot: unavailable)` line. 
Regexp is trying to parse an integer and it is a string.

**Comment by jeffkala** [comment]
closed in #223
